webapp/components/product-builder.js

```diff
@@ -0,0 +1,261 @@
+/**
```

Comment:
FYI, I only scanned the files in this commit (Split product-builder.js out) since I assume its a straight-forward code heist. Lmk if I should take a closer look.

Reply:
Yes it is a simple copy-paste, followed by removing unused imports.